<?xml version="1.0"?>
<launch>
  <arg name="config_path" default="$(find config)/vehicle"/>
  <arg name="sensor_meta"
    default="$(arg config_path)/sensor/calibrated_sensor.pb.txt " />

  <node name="perception_fusion" ns="perception/fusion"
    pkg="perception_fusion" type="perception_fusion_node" output="screen"
    args="--stderrthreshold=0 --v=1
          --sensor_meta_path=$(arg sensor_meta) 
          --multi_sensor_fusion_conf_path=$(arg config_path)/perception/fusion/multi_sensor_fusion.pb.txt" />
</launch>
